src/js/utils/browser.ts

```diff
@@ -81,7 +81,7 @@ export function flashVersion() {
     let flash;
 
     if (plugins) {
-        flash = plugins['Shockwave Flash'];
+        flash = plugins.namedItem('Shockwave Flash');
```

Comment:
is there a lint rule that's yelling about the old way?

Reply:
This was actually throwing a type error. The index type for PluginArrays is 'number', but since we are using a 'string', it was throwing an 'implicitAny' type error